pisek/task_jobs/data/testcase_info.py

```diff
@@ -48,11 +48,31 @@ def mixed(name: str) -> "TestcaseInfo":
     def static(name: str) -> "TestcaseInfo":
         return TestcaseInfo(name, 1, TestcaseGenerationMode.static, False)
 
-    def input_path(self, env: Env, seed: Optional[int] = None) -> TaskPath:
+    def input_path(
+        self, env: Env, seed: Optional[int] = None, solution: Optional[str] = None
```

Comment:
What is the `solution` parameter for?

Reply:
To getting input in directory corresponding to right solution